### ISSUE TITLE
DeepExplainer Shap sums do not add up to model prediction

### ISSUE THREAD

**Issue by maxwellreynolds** [shap/shap#1550]
I am using Deep Shap with a Pytorch model. The model uses a Softmax and does not output a value outside of [0,1] yet the background + expectation is often negative or greater than 1, as shown in a decision plot

![Screen Shot 2020-10-26 at 2 04 06 PM](https://user-images.githubusercontent.com/44449290/97210664-4a0e9380-1794-11eb-838c-6be7fffdcc7e.png)

My understanding was that the sum of all shap values and the expected background value should equal the model's output. Is this incorrect?

**Comment by aionfeehan** [comment]
I'm not entirely sure if this is the problem, but I ran into a similar issue and posted what worked for me in response to the issue here:

https://github.com/slundberg/shap/issues/1678

Hope it's helpful
